ios/vendored/unversioned/@stripe/stripe-react-native/ios/Tests/AddressSheetUtilsTests.swift

```diff
@@ -0,0 +1,273 @@
+//
```

Comment:
we could exclude these test files in expoGoConfig.ts

Reply:
Exclude does not work, seems that `copyVendoredFilesAsync` does not take this `excludeFiles` into account. I just removed manually for now

Reply:
oh yep, the `excludeFiles` on ios was just added last month and only available for *.podspec glob. thanks for checking this.